src/lexer.l

```diff
@@ -165,6 +165,7 @@ oct_esc  [0-7]{1,3}
 "unroll"                { return Parser::make_UNROLL(yytext, driver.loc); }
 "while"                 { return Parser::make_WHILE(yytext, driver.loc); }
 "config"                { return Parser::make_CONFIG(yytext, driver.loc); }
+"opts"                  { return Parser::make_OPTS(yytext, driver.loc); }
```

Comment:
This will need to be added as an extern keyword in the parser.

Reply:
Ah good catch.